Common/File/DiskFree.cpp

```diff
@@ -41,9 +41,11 @@ bool free_disk_space(const Path &path, int64_t &space) {
 	int res = statvfs(path.c_str(), &diskstat);
 
 	if (res == 0) {
+		// Not sure why we're excluding Android here anyway...
```

Comment:
https://github.com/hrydgard/ppsspp/commit/f687613d6875a0cb95eb35ea671e71bcc6a36aa6

IIRC, the flag field wasn't there back then, at least...

-[Unknown]